data/picks/meshsidecar.yaml

```diff
@@ -0,0 +1,12 @@
+---
+description: meshSidecar provides a flexible way to integrate mesh networking capabilities
+  with NixOS services. It allows services to participate in mesh networks without
+  requiring direct modification of the service itself.
```

Comment:
I don't think the whole description will be posted on the site. This might need to change to a single line.

Reply:
Yes it will display.